src/main/java/seedu/address/model/schedule/EventDescription.java

```diff
@@ -36,7 +36,7 @@ public String toString() {
     public boolean equals(Object other) {
         return other == this // short circuit if same object
                 || (other instanceof EventDescription // instanceof handles nulls
-                && value.equals(((EventDescription) other).value)); // state check
+                && value.toLowerCase().equals(((EventDescription) other).value.toLowerCase())); // state check
```

Comment:
can help edit the UG regarding this changes?

Reply:
I will update in a separate PR.